fishtown-analytics/docs.getdbt.com
# Update CSP policy

## What are you changing in this pull request and why?
This PR updates the Content Security Policy headers, specifically `frame-ancestors`, in order to reduce the vulnerability of clickjacking attacks based on our most recent [security report](https://www.notion.so/dbtlabs/Security-Report-Items-Q3-2024-135bb38ebda780b9a1e5e0a82bf49c04).

## Changes
Adds the following frame-ancestors into the content security policy
- `https://*.mutinyhq.com` - Allow experimentation previews within Mutiny dashboard
- `https://*.getdbt.com` - Allows docs to be embedded on sites within our domain

## Preview
https://docs-getdbt-com-git-update-strict-csp-dbt-labs.vercel.app/

## Testing
- Browse through the deploy preview and confirm site functions as expected.
- Confirm no errors are present regarding the changes to the CSP within the browser console.
